src/Data/Object/Simulator.elm

```diff
@@ -74,17 +75,7 @@ computeItemResults db item =
         |> List.map (computeProcessItemResults db)
         |> RE.combine
         |> Result.map
-            (List.foldr
-                (\(Results results) (Results acc) ->
-                    Results
-                        { acc
-                            | impacts = Impact.sumImpacts [ results.impacts, acc.impacts ]
-                            , items = Results results :: acc.items
-                            , mass = Quantity.sum [ results.mass, acc.mass ]
-                        }
-                )
-                emptyResults
-            )
+            (List.foldr accumulateResults emptyResults)
```

Comment:
Nit: This should now be able to fit along `Result.map` on a single line for legibility :)